HerPortal/Views/Home/SupportingDocuments.cshtml

```diff
@@ -20,7 +20,7 @@
     </nav>
 }
 
-<h2 class="govuk-heading-l">Supporting documents</h2>
+<h1 class="govuk-heading-l">Supporting documents</h1>
 <h3 class="govuk-heading-m">Home energy retrofit process</h3>
```

Comment:
I thought about changing this to an H2 as skipping levels isn't ideal, but it would probably be more confusing that way (there would then be two h2 headings with the same text), and Kat's write-up explicitly said to leave the other heading levels as they are.

Reply:
I thought that too, I'm assuming that it is bad, but acceptable in this case.